gossip/service/gossip_service_test.go

```diff
@@ -914,7 +914,7 @@ func TestChannelConfig(t *testing.T) {
 	require.NoError(t, err)
 
 	mockSignerSerializer := &mocks.SignerSerializer{}
-	mockSignerSerializer.SerializeReturns(api.PeerIdentityType(orgInChannelA), nil)
+	mockSignerSerializer.SerializeReturns(orgInChannelA, nil)
```

Comment:
This change is not related to ioutil. I'll defer to @C0rWin since he merged the last change to this file.

Reply:
sorry forgot to add a description

Reply:
indeed, has nothing to do with the declared changes in commit message.
though seems pretty much ok to me.